core/frontend/src/RealityDataSource.ts

```diff
@@ -265,3 +278,104 @@ export class RealityDataSourceProviderRegistry {
     return this._providers.get(name);
   }
 }
+
+/**
+ * Options for creating a Google Photorealistic 3D Tiles reality data source provider.
+ * The caller must provide either an API key or a function that returns an auth token.
+ * @beta
+ */
+export type Google3dTilesProviderOptions = {
+  /** Google Map Tiles API Key used to access Google 3D Tiles. */
+  apiKey: string;
+  getAuthToken?: never;
+  /** If true, the data attributions/copyrights from the Google 3D Tiles will be displayed on screen. The Google Maps logo will always be displayed. Defaults to `true`. */
+  showCreditsOnScreen?: boolean
```

Comment:
isnt allowing users to opt out of this a liability for us? 
when would this be ok?

Reply:
@ggetz Any feedback on your thoughts if a user should be able to disable this feature? (`showCreditsOnScreen`)

Reply:
Lets go ahead and remove this and always show credits. If a user requests to turn this off later we can add this flag back, but I imagine that will be unlikely

Reply:
@aruniverse I don't think there is any harm in exposing this option. The default here is more conservative than what we have in CesiumJS. The opt-in behavior is more or less equivalent to the default CesiumJS behavior.

Reply:
My only hesitation comes from the legal side, but as long as our bases are covered I'm ok. Resolving.